index.html

```diff
@@ -15,8 +15,14 @@
   </head>
    <body>
    <header>
```

Comment:
- [ ] Your current design does not display the background image which is visible on  the header in both the figma desktop and mobile version shown in the images below, please i kindly suggest you add the background image so that your design completely matches the figma template and doing this is important as it shows that you pay good attention to details when designing :+1: :sparkles: 
- figma designs for desktop & mobile header
![h-f-1](https://github.com/kkmanuu/My-Portfolio/assets/89055502/5ccb92f7-c93d-46b3-8119-66f0bee14cb7)
![h-f-2](https://github.com/kkmanuu/My-Portfolio/assets/89055502/68a037e6-28a0-4c1f-8b76-d323040fdc8a)
- your designs for desktop & mobile header
![h-s-1](https://github.com/kkmanuu/My-Portfolio/assets/89055502/c833ed4a-c298-4100-8af9-e9633c5b69f6)
![h-s-2](https://github.com/kkmanuu/My-Portfolio/assets/89055502/6e769312-4fee-43ee-9e34-dbe340f18778)

Reply:
I have made the necessary changes to the design, and it has displayed the background image correctly.